src/components/interactive-map.js

```diff
@@ -347,6 +350,11 @@ export default class InteractiveMap extends PureComponent<InteractiveMapProps, S
     return event;
   }
 
+  _onLoad = () => {
+    this.setState({stylesLoaded: true});
+    this.props.onLoad();
```

Comment:
Need to forward the event object to the callback.

Reply:
I couldn't find documentation what the event object here is, but you're right that there is an event being triggered here.

Interestingly enough, the [typescript type definitions here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-map-gl/index.d.ts#L43) also aren't aware there's an event object.

Will fix though, thanks for catching this!